pkg/vadmin/revive_db_vc.go

```diff
@@ -67,5 +67,12 @@ func (v *VClusterOps) genReviveDBOptions(s *revivedb.Parms, certs *HTTPSCerts) *
 	opts.Cert = certs.Cert
```

Comment:
During revive we do a prepass where we download the cluster_config.json (see describe_db_vc.go). It calls the revive API under the covers with a `DisplayOnly` option set. Do we need the restore options for that? It might depend on whether cluster_config.json is versioned in some way. For instance, if change the cluster_config.json by adding/removing a node, if we then restore to a point in time before the scaling operation, what nodes do we see? We may want to investigate this as a follow-on issue if there isn't a simple answer.

Reply:
I heard that the cluster_config.json is versioned for restore points. So, I think we need to include the restore options in describe_db_vc.go as well.

Reply:
I thought the restore point parameters are actually only used in `makeNMALoadRemoteCatalogOp`, which is called inside `produceReviveDBInstructions`, which will not be invoked if we have `DisplayOnly` set

Reply:
But it's a good idea to also test the scenario where we scale out or scale in after saving restore points and then restore

Reply:
Lots of questions here. I opened VER-91328 so that we can continue the investigation next sprint. No need to do anything about this in the current PR.